docs/samples/tensorflow/README.md

```diff
@@ -57,4 +57,49 @@ Expected Output
     ]
 * Connection #0 to host 34.83.190.188 left intact
 }%
+```
+
+If you stop making requests to the application, you should eventually see that your application scales itself back down to zero. Watch the pod until you see that it is `Terminating`. This should take approximately 90 seconds.
+
+    ```
+    kubectl get pods --watch
+    ```
+
+    Note: To exit the watch, use `ctrl + c`.
+
+## Canary Rollout
+
+To test a canary rollout, you can use the tensorflow-canary.yaml 
+
+Apply the CRD
+```
+kubectl apply -f tensorflow-canary.yaml 
+```
+
+To verify if your traffic split percenage is applied correctly, you can use the following command using the [Knative CLI](#knative-cli)
```

Comment:
Thanks to our new printing, this is now available in:
```kubectl get kfservices```

Reply:
Ok will add it as default option to use.

Reply:
Done